Include/object.h

```diff
@@ -524,6 +524,37 @@ static inline void _Py_DECREF(
 #  define Py_DECREF(op) _Py_DECREF(_PyObject_CAST(op))
 #endif
 
+static inline void
+_Py_DECREF_SPECIALIZED(PyObject *op, const destructor destruct)
```

Comment:
It would be safer to keep `_Py_DECREF_*()` exclusive for internal use, until their usages are well established.

Reply:
Thanks -- moved to include/internal/pycore_object.h, hopefully that's appropriate